psi4/src/psi4/libfunctional/LibXCfunctional.cc

```diff
@@ -213,22 +213,10 @@ double LibXCFunctional::query_density_cutoff() {
 void LibXCFunctional::set_omega(double omega) {
     omega_ = omega;
     user_omega_ = true;
-    if (xc_func_name_ == "XC_GGA_X_WPBEH") {
-        xc_functional_->cam_omega = omega;
-    } else if (xc_func_name_ == "XC_GGA_X_HJS_PBE") {
-        xc_functional_->cam_omega = omega;
-    } else if (xc_func_name_ == "XC_HYB_GGA_XC_LRC_WPBEH") {
-        xc_functional_->cam_omega = omega;
-    } else if (xc_func_name_ == "XC_HYB_GGA_XC_WB97X") {
-        xc_functional_->cam_omega = omega;
-    } else if (xc_func_name_ == "XC_HYB_GGA_XC_WB97") {
-        xc_functional_->cam_omega = omega;
-    } else if (xc_func_name_ == "XC_HYB_GGA_XC_WB97X_V") {
-        xc_functional_->cam_omega = omega;
-    } else if (xc_func_name_ == "XC_HYB_GGA_XC_WB97X_D") {
-        xc_functional_->cam_omega = omega;
-    } else if (xc_func_name_ == "XC_HYB_MGGA_X_M11") {
-        xc_functional_->cam_omega = omega;
+    if ((xc_func_name_ == "XC_GGA_X_WPBEH") || (xc_func_name_ == "XC_GGA_X_HJS_PBE") || (xc_func_name_ == "XC_HYB_GGA_XC_LRC_WPBEH") ||
+        (xc_func_name_ == "XC_HYB_GGA_XC_WB97X") || (xc_func_name_ == "XC_HYB_GGA_XC_WB97") || (xc_func_name_ == "XC_HYB_GGA_XC_WB97X_V") ||
+        (xc_func_name_ == "XC_HYB_GGA_XC_WB97X_D") || (xc_func_name_ == "XC_HYB_MGGA_X_M11")) {
```

Comment:
```suggestion
    if xc_func_name_ in ["XC_GGA_X_WPBEH", "XC_GGA_X_HJS_PBE", "XC_HYB_GGA_XC_LRC_WPBEH", "XC_HYB_GGA_XC_WB97X", "XC_HYB_GGA_XC_WB97", "XC_HYB_GGA_XC_WB97X_V", "XC_HYB_GGA_XC_WB97X_D", "XC_HYB_MGGA_X_M11"]:
```

Reply:
in C++? I thought `std::find` was the shortest (https://stackoverflow.com/a/24139474) but hadn't looked it up yet.

Reply:
:clown_face: nevermind

Reply:
Too much Python lately